codice/ddf
# [2.13.x] DDF-4147 Modified UI installer and migration:import to restart Solr as well as DDF

#### What does this PR do?
It modified the installer UI such that when comes time to restart DDF, it actually does it not through karaf reboot behavior but takes advantage of the restart logic defined inside the `ddf` and `ddf.bat` scripts. This has the advantages that the ddf scripts will actually restart Solr as well if it needs to.

It also modified migration:import to do the same thing.

#### Who is reviewing it? 
@ethantmanns 
@jhunzik 
@tyler30clemens 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.
@figliold
@tbatie

#### How should this be tested?
1- Unzip DDF and proceed through the UI installer
2- Click restart
3- Verify that Solr is being shutdown and then restarted in addition to DDF
4- Perform a migration:export
5- Perform a migration:import
6- Same as step 3

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4147](https://codice.atlassian.net/browse/DDF-4147)

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
